pyproject.toml

```
@@ -25,7 +25,7 @@ classifiers = [
 ]
 requires-python = ">=3.9,<3.13"
 dependencies = [
-    "datasets>=2.19.0, <4.0.0",
+    "datasets==4.0.0",
```

We don't support datasets `v4`. I've tested on this version
```suggestion
    "datasets<4.0.0",
```

---

I think with `pull_request` token from your repo is taken, but you don't have it. To fix this `pull_request_target` should be used probably

---

@Samoed Ok, i changed to `pull_request_target`. I added my token to my repo as well.